lib/google/gapic/runner/proto_compiler_runner.rb

```diff
@@ -58,25 +60,32 @@ def request
         end
         private :request
 
-        # Private.
-        # The path where the templates are located.
-        # TODO: Support custom paths here via env variable.
-        # @return [String]
-        def template_path
-          relative = File.join(*['..']*5, 'templates')
-          File.expand_path(relative, __FILE__)
-        end
-        private :template_path
-
         # Private.
         # The template provider for this run.
-        # @return [Google::Gapic::Generator::TemplateProvider]
+        # @return [ActionController::Base]
         def template_provider
```

Comment:
Can we come up with a better name for this?

Reply:
Changing to `controller` to reflect the actual type.

Reply:
FWIW, I'm fine keeping the current name for now.

Reply:
@blowmage I already merged, but feel free to change it back.